backend/src/main/kotlin/rocks/didit/sefilm/domain/Swish.kt

```diff
@@ -23,7 +23,7 @@ data class SwishDataDTO(
 
     fun generateUri(): URI {
         val asString = jacksonObjectMapper().writeValueAsString(this)
-        val encodedData = URLEncoder.encode(asString, StandardCharsets.UTF_8.displayName())
+        val encodedData = URLEncoder.encode(asString, StandardCharsets.UTF_8.displayName()).replace("+", "%20")
```

Comment:
Yup, we can use Guavas implementation that "correctly" encodes. That is how I do it in swosh.me